embuilder.py

```diff
@@ -157,6 +158,10 @@ def get_system_tasks():
   return system_libraries, system_tasks
 
 
+def should_use_ninja():
+  return os.environ.get('EMCC_USE_NINJA', 0)
```

Comment:
Can you just do `from system_libs import USE_NINJA` instead?

Reply:
Yeah that works. If it's going to be exported from system_libs, should we make it a function there, instead of a global?

Reply:
(I went ahead and did that)